python/dgl/graphbolt/utils/sample_utils.py

```diff
@@ -10,7 +10,8 @@ def unique_and_compact_node_pairs(
     node_pairs: Union[
         Tuple[torch.Tensor, torch.Tensor],
         Dict[Tuple[str, str, str], Tuple[torch.Tensor, torch.Tensor]],
-    ]
+    ],
+    unique_dst_nodes=None,
```

Comment:
What's the type of the unique_dst_nodes?

Reply:
Added